lib/db/mysql.js

```diff
@@ -788,6 +788,13 @@ module.exports = function (log, error) {
     return this.readFirstResult(GET_SECONDARY_EMAIL, [emailBuffer.toString('utf8')])
   }
 
+  // Get : account
+  // Values : email = $1
```

Comment:
Not sure if it's important but the standard pattern for these comments on reads seems to be more like:

```js
// Select : accounts a, emails e
// Fields : a.uid, a.email, ...
// Where : a.uid = ... AND e.isPrimary = ...
```